lib/msf/base/sessions/ssh_command_shell_bind.rb

```diff
@@ -116,27 +116,6 @@ def close_write
         @ssh_channel.eof!
       end
 
-      #
```

Comment:
This is moved into the new `Rex::Post::Channel::StreamAbastraction` module to reduce repetition.